scripts/segmentation_HALO-20240906a.md

```diff
@@ -0,0 +1,363 @@
+plt.plot(meteor_track.lon, meteor_track.lat, c="C4", ls=":", label="METEOR track")
+plt.xlabel("longitude / °")
+plt.ylabel("latitude / °")
+plt.legend();
+```
+
+## Interactive plots
+
+```python
+ds["alt"].hvplot()
+```
+
+```python
+ds["roll"].hvplot()
+```
+
+```python
+ds["heading"].hvplot()
+```
+
+## Segments
+
+defined as a tuple of time slice (`start`, `end`) , segment `kind`, `name`, `remarks`.
+
+* in case of irregularities within a circle, 1 sec before the first and after the last sonde are chosen as start and end times
+* use the list of `remarks` to state any deviations, also with respective times
+
+Alternatively, you can also define the segments as dictionaries which also allows to add further attributes to single segments, e.g. a `radius` to a `circle` segment. At the end of the following code block all segments will be normalized by the `parse_segments` function.
+
+```python
+sl1 = (
+    slice("2024-09-06T10:43:06", "2024-09-06T11:05:05"),
+    ["straight_leg", "ascent"],
+    "leg 1 climbing and heading west",
+    ["irregularity: roll angle deviation at 11:00:54"],
+)
+
+sl2 = (
+    slice("2024-09-06T11:09:50", "2024-09-06T11:39:28"),
+    ["straight_leg"],
+    "leg2 heading south",
+)
+
+sl3 = (
+    slice("2024-09-06T11:39:28", "2024-09-06T11:47:55"),
+    ["straight_leg", "ascent"],
+    "leg 3 heading south while climbing",
+)
+
+
+sl4 = (
+    slice("2024-09-06T11:47:55", "2024-09-06T12:18:27"),
+    ["straight_leg"],
+    "leg 4 heading south",
+)
+
+sl5 = (
+    slice("2024-09-06T12:20:04", "2024-09-06T12:26:40"),
+    ["straight_leg"],
+    "southern leg 5",
+)
+
+sl6 = (
+    slice("2024-09-06T12:28:00", "2024-09-06T12:37:19"),
+    ["straight_leg"],
+    "southern leg 6",
+)
+
+sl7 = (
+    slice("2024-09-06T12:44:14", "2024-09-06T12:58:09"),
+    ["straight_leg"],
+    "southern leg 7",
+)
+
+sl8 = (
+    slice("2024-09-06T13:01:59", "2024-09-06T13:07:03"),
+    ["straight_leg"],
+    "southern leg 8 meeting METEOR",
+)
+
+sl9 = (
+    slice("2024-09-06T13:11:29", "2024-09-06T13:19:16"),
+    ["straight_leg"],
+    "southern leg 9",
+)
+
+sl10 = (
+    slice("2024-09-06T13:19:36", "2024-09-06T14:07:18"),
+    ["straight_leg"],
+    "southern leg 10",
+)
+
+sl11 = (
+    slice("2024-09-06T14:17:58", "2024-09-06T14:40:13"),
+    ["straight_leg"],
+    "southern leg 11",
+)
+
+sl12 = (
+    slice("2024-09-06T14:42:58", "2024-09-06T14:54:11"),
+    ["straight_leg"],
+    "leg 12 heading north",
+)
+
+sl13_1 = (
+    slice("2024-09-06T14:59:12", "2024-09-06T15:05:22"),
+    ["straight_leg", "ascent"],
+    "climbing while heading north",
+)
+
+sl13_2 = (
+    slice("2024-09-06T15:05:22", "2024-09-06T15:20:20"),
+    ["straight_leg"],
+    "leg crossing circle",
+)
+
+c1 = (
+    slice("2024-09-06T15:23:12", "2024-09-06T16:21:35"),
+    ["circle"],
+    "circle west",
+)
+c1_1 = (
+    slice("2024-09-06T16:21:35", "2024-09-06T16:41:20"),
+    [],
+    "continued circling",
+)
```

Comment:
I would remove this one for now until we have clarified how we want to deal with circles larger than 360deg.
```suggestion
# c1_1 = (
#     slice("2024-09-06T16:21:35", "2024-09-06T16:41:20"),
#     [],
#     "continued circling",
# )
```